### ISSUE TITLE
main.profileConfig: panic: assignment to entry in nil map

### ISSUE THREAD

**Issue by pda** [99designs/aws-vault#32]
Added a second access key:

```
pda@paulbook ~ ⸩ aws-vault add pda
Enter Access Key ID: AKI…PFQ
Enter Secret Access Key:
Added credentials to profile "pda" in vault
```

`aws-vault exec` crashes attempting to use it:

```
pda@paulbook ~ ⸩ aws-vault exec pda -- env
panic: assignment to entry in nil map

goroutine 1 [running]:
main.profileConfig(0x7fff5fbffb2f, 0x3, 0x4f04a60, 0x0, 0x0)
        /Users/pda/code/go/src/github.com/99designs/aws-vault/exec.go:104 +0x2eb
main.ExecCommand(0xc8200848c0, 0xc820084910, 0xc820084960, 0x4575ac0, 0x7fff5fbffb2f, 0x3, 0x7fff5fbffb36, 0x3, 0x0, 0x0, ...)
        /Users/pda/code/go/src/github.com/99designs/aws-vault/exec.go:53 +0x54f
main.main()
        /Users/pda/code/go/src/github.com/99designs/aws-vault/main.go:102 +0x2983
```

Currently using locally-compiled `aws-vault` from https://github.com/99designs/aws-vault/pull/30 which is v2.3.1 + small patch.

**Comment by pda** [comment]
Caused by missing `[profile pda]` in `~/.aws/config`

**Comment by lox** [comment]
Fixed in 2eaa66639fe6bc526dc85eca3d3b04197e232473

**Comment by pda** [comment]
:+1:
